### ISSUE TITLE
{% static ... as img_path %} doesn't work with arrays

### ISSUE THREAD

**Issue by killianarts** [wrabit/django-cotton#100]
I have some code like this:

```django
{% static 'path/to/image/one' as img_one %}
{% static 'path/to/image/two' as img_two %}
<c-slider :images="[{{ img_one }}, {{ img_two }}]">
```

However, the image paths aren't being added to the array.

```django
<c-slider an_image="{{ img_one }}">
```

Works as expected.

**Comment by wrabit** [comment]
@killianarts I will check a bit later but one thought is:

```html
<c-slider :images="[
    '{% static 'path/to/image/one'%}',
    '{% static 'path/to/image/two'%}'
]">
```

(not tested)

**Comment by wrabit** [comment]
@killianarts Addressed in the latest release, v0.9.33. Please note you'll still need to wrap your list values in quotes in order for the component to receive a list of strings.

```html
<c-slider :images="['{{ img_one }}', '{{ img_two }}']">
```

**Comment by killianarts** [comment]
Great! Thank you!
